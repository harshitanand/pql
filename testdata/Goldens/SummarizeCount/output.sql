WITH "__subquery0" AS (SELECT "Directory" AS "Directory", endsWith("FileName", '_test.go') AS "IsTest", count() AS "Files" FROM "SourceFiles" GROUP BY "Directory", endsWith("FileName", '_test.go'))
SELECT * FROM "__subquery0" ORDER BY "Directory" ASC NULLS FIRST, "IsTest" ASC NULLS FIRST;
