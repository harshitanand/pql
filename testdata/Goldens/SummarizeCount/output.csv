.,0,1
.,1,3
parser,0,4
parser,1,2
